### PR TITLE
ContentRoot Fix

### PR DESCRIPTION
**Pull request by shanselman**
This can't work, so I added the ContentRoot calls to make sure it runs.

### CODE REVIEW

File: samples/Localization.StarterWeb/Startup.cs [hunk 1]
```diff
@@ -15,6 +15,7 @@
 using Microsoft.AspNetCore.Localization;
 using System.Globalization;
 using Microsoft.Extensions.Options;
+using System.IO;
```
Comment (davidfowl) [inline samples/Localization.StarterWeb/Startup.cs]: nit: Sort usings

File: samples/Localization.StarterWeb/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -54,7 +54,7 @@
 
         <script src="//ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.4.min.js"></script>
         <script src="//ajax.aspnetcdn.com/ajax/bootstrap/3.3.5/bootstrap.min.js"></script>
-        <script src="~/js/site.min.js" asp-append-version="true"></script>
+        <script src="~/js/site.js" asp-append-version="true"></script>
```
Comment (muratg) [inline samples/Localization.StarterWeb/Views/Shared/_Layout.cshtml]: Why not minimized version?

Comment (ryanbrandenburg) [inline samples/Localization.StarterWeb/Views/Shared/_Layout.cshtml]: The minimized version is empty and the unminified version is only like 3 lines long in the first place. @shanselman could you remove site.min.js while you're at it?

File: samples/Localization.StarterWeb/Startup.cs [hunk 3]
```diff
@@ -1,19 +1,20 @@
 using System;
```
Comment (ryanbrandenburg) [inline samples/Localization.StarterWeb/Startup.cs]: While we're messing around with the usings we may as well remove the couple unused ones in here.

**Comment by ryanbrandenburg** [comment]
Other than some small nit-picks this looks like a :shipit:. Thanks for catching it!

**Comment by Eilon** [comment]
@ryanbrandenburg can you make sure this PR is still valid and merge (or fix if needed)?

**Comment by ryanbrandenburg** [comment]
Merged in 802ca6b75.

**Pull request closed without merging**